Examples/Applications/SkyboxViewer/index.js

```diff
@@ -189,7 +189,8 @@ function createVisualization(container, mapReader) {
     updateSkybox(allPositions[nextIdx]);
   }
 
-  if (enableVR && vtkDeviceOrientationToCamera.isDeviceOrientationSupported()) {
+  if (enableXR && navigator.xr.isSessionSupported('immersive-vr')) {
```

Comment:
@floryst @martinken I'd like to propose that we remove the `enableVR`/`enableXR` flag entirely from SkyboxViewer.
- Subverts typical VR design by immediately trying to render a VR scene rather than presenting the user with a button to opt-in;
- Different design pattern using two different renderers, as opposed to two passes over the same renderer slightly translated;
- Relies on `vtkRadialDistortionPass` which doesn't seem to be used anywhere else and isn't necessary for at least the majority of VR viewers (potentially Cardboard?)

If preferred we can ignore the flag for now and then in a subsequent change move this functionality from the `SkyboxViewer` example into a dedicated example demonstrating how vtk.js *could* be used for a dedicated Cardboard viewing experience.

Reply:
I agree that VR should have a send action rather than dropping directly into it. My guess is that it was like this for some early demo.

Reply:
Removed in fixup commit